features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -1780,6 +1782,15 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }, [ initialValues ]
     );
 
+    useEffect(
+        () => {
+            setCurrentBindingType(initialValues?.accessToken?.bindingType
+                ? initialValues.accessToken.bindingType
+                : isFAPIApplication ? SupportedAccessTokenBindingTypes.CERTIFICATE
+                    : metadata?.accessTokenBindingType?.defaultValue
+                ?? SupportedAccessTokenBindingTypes.NONE);
+        }, [ initialValues, metadata ]
```

Comment:
Let's use exact attribute in the dependency array, so it will prevent unnecessary triggers.

```suggestion
        }, [ initialValues?.accessToken?.bindingType, metadata?.accessTokenBindingType?.defaultValue ]
```